inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/meta/ZookeeperWatcher.java

```diff
@@ -38,10 +39,16 @@
 import org.apache.curator.framework.recipes.cache.PathChildrenCacheListener;
 import org.apache.curator.framework.state.ConnectionState;
 import org.apache.curator.framework.state.ConnectionStateListener;
+import org.apache.inlong.sort.configuration.Configuration;
+import org.apache.inlong.sort.configuration.Constants;
+import org.apache.inlong.sort.meta.MetaManager.DataFlowInfoListener;
+import org.apache.inlong.sort.util.ZooKeeperUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ZookeeperWatcher implements AutoCloseable, UnhandledErrorListener {
+import com.google.common.base.Preconditions;
+
+public class ZookeeperWatcher implements AutoCloseable, UnhandledErrorListener, MetaWatcher {
```

Comment:
ZookeeperWatcher is a basic util class wrapping zk related operations. It should not implement MetaWatcher.
Use another class ZookeeperMetaWatcher which implements MetaWatcher instead.

Reply:
ok, add another class ZookeeperMetaWatcher  which implements MetaWatcher, and change ZookeeperWatcher  to ZookeeperWatcherUtils.